smoke-tests/apps/Micrometer1_5/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/MicrometerDisabledTest.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.applicationinsights.smoketest;
+
+import static com.microsoft.applicationinsights.smoketest.EnvironmentValue.JAVA_8;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+@Environment(JAVA_8)
+@UseAgent("disabled_applicationinsights.json")
+class MicrometerDisabledTest {
+
+  @RegisterExtension static final SmokeTestExtension testing = SmokeTestExtension.create();
+
+  @Test
+  @TargetUri("/test")
+  void doMostBasicTest() throws Exception {
+    Telemetry telemetry = testing.getTelemetry(0);
+
+    assertThat(telemetry.rd.getName()).isEqualTo("GET /test");
+    assertThat(telemetry.rd.getSuccess()).isTrue();
+
+    // sleep a bit and make sure no micrometer metrics are reported
+    Thread.sleep(10000);
```

Comment:
To make the test faster, would it be possible to check if assertions just after are successful every 1 second for example, and and this up to 10 seconds maximum?

Reply:
this is ensuring that _no_ metrics are reported, so verifying that no metrics are reported after just 1 second is not enough

Reply:
The test could fail if metrics are reported after 2 seconds for example (this could be checked every second for example)?

Reply:
that's true, I'm not too worried about test slowness when it comes to failures, but if you would like, we should do it across all of our smoke tests (it looks like this pattern is used in ~20 places)

Reply:
Yes, it could be great to do it across all of our smoke tests , or start to do this for this test in this PR (and apply the same pattern for other smoke tests in another PR). In one previous project, I have applied this pattern and the reduction on build duration was significant.